docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -159,11 +159,12 @@ This event will be sent when session closes successfully.
 #### Attributes
 
 | Name               | Type                 | Description                                                                              |
-| :----------------- | :------------------- | :--------------------------------------------------------------------------------------- |
+|:-------------------|:---------------------|:-----------------------------------------------------------------------------------------|
 | room_id            | `string`             | 100ms assigned room id <br/><br/> _Example: 5f9edc6ac238215aec2312df_                    |
 | room_name          | `string`             | Room name provided when creating the room <br/><br/> _Example: Test Room _               |
 | session_id         | `string`             | 100ms assigned id to identify the session <br/><br/> _Example: 5f9edc6bd238215aec7700df_ |
 | template_id        | `string`             | Template ID of the room <br/><br/> _Example: 66112497abcd52312556c4gg_                   |
+| session_duration   | `number`             | Duration the user spent in the room in seconds <br/><br/> _Example: 36000 _              |
```

Comment:
@atul107 type should be integer? similarly in all other occurrences of type as number.

Reply:
changed to int